src/Engine/ProtoAssociative/CodeGen_SSA.cs

```diff
@@ -488,7 +441,6 @@ private void DFSEmitSSA_AST(AssociativeNode node, Stack<AssociativeNode> ssaStac
 
                 var bnode = AstFactory.BuildAssignment(leftNode, rightNode);
                 bnode.isSSAAssignment = isSSAAssignment;
-                bnode.IsInputExpression = astBNode.IsInputExpression;
```

Comment:
This function is skipped from being called for ASTs that are "input" expressions.

Reply:
If we ever intend to make SSA form work with this input optimization maybe we should leave this in, commented out with a TODO?